src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -20,6 +21,13 @@ public class MonitoringConfiguration : ConfigurationCategory
         [ConfigurationEntry("Monitoring.Cpu.Exec.Arguments", ConfigurationEntryScope.ServerWideOnly, isSecured: true)]
         public string CpuUsageMonitorExecArguments { get; set; }
 
+        [Description("The minimum interval between measures to calculate the disk stats")]
+        [DefaultValue(1)]
+        [MinValue(1)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry("Monitoring.MinDiskStatsInterval", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
`InSeconds` should be added.
But do we _want_ to limit it to sec granularity?

Reply:
@gregolsky, what do you think?

Reply:
@gregolsky, what do you think?